src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -156,51 +145,20 @@ void givenUsedApplicationIdFromAdminUserRegisterRequest_whenApplicationEntitySta
 
         // Then
         Assertions.assertThrows(
-                AysAdminUserRegisterApplicationCodeNotValidException.class,
-                () -> adminUserRegisterService.register(mockAdminUserRegisterRequest)
+                AysAdminUserRegisterApplicationNotExistByIdAndStatusException.class,
+                () -> adminUserRegisterService.completeRegistration(applicationId, mockAdminUserRegisterApplicationCompleteRequest)
         );
 
         Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Ekledim.

Reply:
✔ Bu kısmı **Resolve Conversation** edebiliriz.